src/containers/HubiiApiHoc/selectors.js

```diff
@@ -1,58 +1,75 @@
-import { createSelector } from 'reselect';
+import { createSelector, createSelectorCreator, defaultMemoize } from 'reselect';
 import { fromJS, List } from 'immutable';
 import BigNumber from 'bignumber.js';
 
-import { makeSelectBlockHeight } from 'containers/EthOperationsHoc/selectors';
 import { createDeepEqualSelector } from 'utils/selector';
 
 /**
  * Direct selector to the hubiiApiHoc state domain
  */
 const selectHubiiApiHocDomain = (state) => state.get('hubiiApiHoc');
 
-/**
- * Other selectors
- */
-const makeSelectTransactions = () => createSelector(
-  selectHubiiApiHocDomain,
-  (hubiiApiHocDomain) => hubiiApiHocDomain.get('transactions') || fromJS([])
+export const createTransactionsSelector = createSelectorCreator(
+  defaultMemoize,
+  (currentArray, previousArray) => {
+    let changed = false;
+    currentArray.keySeq().forEach((address) => {
+      if (!currentArray.getIn([address, 'transactions']) || !previousArray.getIn([address, 'transactions'])) {
+        return;
```

Comment:
No return value, just undefined?

Reply:
It is a loop through an array to check if the array has been updated or not. The `return` in this line is to skip the check for the current address if it doesn't meet the sanity condition.